src/OMSimulatorLib/Connection.h

```diff
@@ -39,6 +39,44 @@
 
 #include <string>
 
+namespace oms3
+{
+  /// **
+  // * \brief Connection
+  // * /
+  class Connection : protected oms3_connection_t
+  {
+  public:
+    Connection(const oms3::ComRef& conA, const oms3::ComRef& conB);
+    ~Connection();
+
+    // methods to copy the object
+    Connection(const Connection& rhs);
+    Connection& operator=(const Connection& rhs);
+
+    oms_status_enu_t exportToSSD(pugi::xml_node& root) const;
+
+    const oms3::ComRef getSignalA() const {return oms3::ComRef(conA);}
+    const oms3::ComRef getSignalB() const {return oms3::ComRef(conB);}
+
+    const oms2::ssd::ConnectionGeometry* getGeometry() const {return reinterpret_cast<oms2::ssd::ConnectionGeometry*>(geometry);}
+    void setGeometry(const oms2::ssd::ConnectionGeometry* newGeometry);
+
+    bool isEqual(const oms3::Connection& connection) const;
+    bool isEqual(const oms3::ComRef& signalA, const oms3::ComRef& signalB) const;
+
+    void describe();
```

Comment:
@robbr48 @adeas31 Do you think we should support the `describe()` functionality? I think we can just remove that.

Reply:
I have no strong opinion. For me its fine to remove it.